app/javascript/components/pages/__tests__/FateNew.test.js

```diff
@@ -3,13 +3,27 @@ import Enzyme, { shallow } from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
 import FateNew from '../FateNew'
 
+
 Enzyme.configure({ adapter: new Adapter() })
 
 describe('when FateNew renders', () => {
-    test('should display fatenew heading', () => {
+    test('should display a form', () => {
         const fateNew = shallow(<FateNew/>)
-        const fateNewHeading = fateNew.find("h1")
-      expect(fateNewHeading.text()).toEqual("This is the FateNew page")
+        const form = fateNew.find("Form")
+        expect(form.length).toEqual(1)
     })
-
 })
+
+// fdescribe('when click on submit redirected to index', () => {
```

Comment:
Do these need to be fixed or removed?

Reply:
they are stretch goals, we got a bit to far into testing and tried testing if the page would re direct, we were informed thats not as important as the app working, so they've been iceboxed for now.

Reply:
ahh.. they should be removed then. You can always grab them from the commit later if you want. That's what git is for! woot.